examples/tutorial/tutorial_notebooks/introduction/exercise_1.ipynb

```diff
@@ -210,7 +744,7 @@
  ],
```

Comment:
suggestion: instead of a plain text instructing users to move onto the next exercise, consider adding a link to the next exercise directly.

Reply:
@jeffreykennethli I added hyperlink.